ui/lib/core/addon/components/form-field.hbs

```diff
@@ -17,6 +17,21 @@
           data-test-input-group={{@attr.name}}
           as |G|
         >
+          <G.Legend data-test-form-field-label={{@attr.name}}>{{this.labelString}}</G.Legend>
+          {{#if this.helpTextString}}
+            <G.HelperText data-test-help-text={{@attr.options.helpText}}>{{this.helpTextString}}</G.HelperText>
+          {{/if}}
+          {{#if @attr.options.subText}}
+            <G.HelperText data-test-help-text={{@attr.options.subText}}>
+              {{@attr.options.subText}}
+              {{#if @attr.options.docLink}}
+                <Hds::Link::Inline data-test-doc-link={{@attr.options.docLink}} @href={{doc-link @attr.options.docLink}}>
```

Comment:
`<DocLink` is technically deprecated, so using Hds::Link here. Note - currently no radio edit types have a doc link passed

Reply:
Just noticed I need to pass `@isHrefExternal={{true}`